src/Microsoft.AspNetCore.Sockets.Common/ContentTypes.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Sockets
+{
+    public static class ContentTypes
+    {
+        public static readonly string TextContentType = "application/vnd.microsoft.aspnetcore.endpoint-messages.v1+text";
```

Comment:
This could be simpler now that it's at a higher level. It could live in SignalR and be `application/vnd.microsoft.signalr.v1+json`, and similar for ProtoBuf.

We'd need to expose `Content-Type` in the metadata as well. Also, we need to think about how protocols are negotiated in this model. Is it specified in a header/query string value? `Accept` headers? We can do standard content negotiation for Long Polling, but WebSockets and SSE have fixed header sets and we can't apply request headers easily.

If you want to file an issue for this and do it in a separate PR that's fine, but we should tidy this all up.